src/main/java/com/example/demo/entity/Character.java

```diff
@@ -0,0 +1,335 @@
+package com.example.demo.entity;
+
+import java.util.Set;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+
+
+@Entity
+@Table(name = "Character")
+public class Character {
+
+   @Id
+   @GeneratedValue
+   private Long id;
+
+   private Integer age;
+   private Integer height;
+   private String  biography;
+   private String  name;
+
+   private Integer strength;
+   private Integer dexterity;
+   private Integer constitution;
+   private Integer intelligence;
+   private Integer wisdom;
+   private Integer charisma;
+   private Integer armor;
+   private Integer addiction;
+   private Integer speed;
+   private Integer perception;
+
+
+   @ManyToMany
+   @JoinTable(
+        name = "character_title",
+        joinColumns = @JoinColumn(name = "characterId"),
+        inverseJoinColumns = @JoinColumn(name = "titleId")
+    )
+   private Set<Title> titles;
+
+
+   @ManyToOne
```

Comment:
Miss many of attributs for the character sheet game to add later. But for the presentation it's ok